InclusionBot.md

```diff
@@ -73,9 +73,11 @@ approves your changes, they will be merged and become active in the bot.
-| Goddamn, goddam, goddamnit, goddammit            | awful, terrible, wretched                                                                     | This term can be offensive or triggering to some religious traditions because it implies a deity cursing or reproaching people. We want to show respect towards all religious traditions.                                                                                                                                                                                                                              |
-| jesus fucking, fucking christ                    | Wow, oh no, oh my                                                                             | We want to show respect towards all religious traditions.                                                                                                                                                                                                                                                                                                                                                              |
-| mumbo jumbo                                      | jargon, confusing language                                                                    | The term "mumbo jumbo" was used to stereotype African languages and cultures that didn't make sense to European colonizers. <br><br> See [Here are some commonly used terms that actually have racist origins](https://abcnews.go.com/Politics/commonly-terms-racist-origins/story?id=71840410) for more.                                                                                                                                                                                                                                                                                                                                                            |
+| Goddamn, goddam, goddamnit, goddammit            | awful<br>terrible<br>wretched                                                                 | This term can be offensive or triggering to some religious traditions because it implies a deity cursing or reproaching people. We want to show respect towards all religious traditions.                                                                                                                                                                                                                              |
```

Comment:
concern: Not directly related to the PR, but these are miscategorized. These religion-based lines are under the **Racist** table.

change (techincally scope creep): Can we add a catch-all table and move the religion ones there?

Reply:
That's a good idea! I'll create an issue for it.